src/Lykke.Service.ClientAccountRecovery/Modules/ServiceModule.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Autofac;
+using AzureStorage;
+using AzureStorage.Tables;
+using Lykke.Common.Log;
+using Lykke.Service.ClientAccount.Client;
+using Lykke.Service.ClientAccountRecovery.AzureRepositories;
+using Lykke.Service.ClientAccountRecovery.Core;
+using Lykke.Service.ClientAccountRecovery.Core.Services;
+using Lykke.Service.ClientAccountRecovery.Services;
+using Lykke.Service.ClientAccountRecovery.Settings;
+using Lykke.Service.ConfirmationCodes.Client;
+using Lykke.Service.Kyc.Abstractions.Services;
+using Lykke.Service.Kyc.Client;
+using Lykke.SettingsReader;
+
+namespace Lykke.Service.ClientAccountRecovery.Modules
+{
+    public class ServiceModule : Module
+    {
+        private readonly IReloadingManager<AppSettings> _settings;
+
+        public ServiceModule(IReloadingManager<AppSettings> settings)
+        {
+            _settings = settings;
+        }
+
+        // NOTE: you can remove it if you don't need to use IServiceCollection extensions to register service specific dependencies
+
+
+        protected override void Load(ContainerBuilder builder)
+        {
+
+
+            builder.Register(c => _settings.Nested(n => n.ClientAccountRecoveryService.RecoveryConditions));
+
+
+            builder.RegisterType<HealthService>()
+                .As<IHealthService>()
+                .SingleInstance();
+
+            builder.RegisterType<StartupManager>()
+                .As<IStartupManager>();
```

Comment:
General comments to all registrations:
- use .AsImpementedInterfaces instead of .As<>
- review lifetime scopes, most of them can be made single instance

Reply:
1. For what? I'm sure what interface I want to register. I don't need all other interface if any.
2. Again, for what?